test/conversion_test.rb

```diff
@@ -0,0 +1,206 @@
+require "test_helper"
+
+class Measured::ConversionTest < ActiveSupport::TestCase
+  setup do
+    @conversion = Measured::Conversion.new
+  end
+
+  test "#base sets the base unit" do
+    @conversion.set_base :m, aliases: [:metre]
+    assert_equal ["m", "metre"], @conversion.base_unit.names
+  end
+
+  test "#base doesn't allow a second base to be added" do
+    @conversion.set_base :m, aliases: [:metre]
+
+    assert_raises Measured::UnitError do
+      @conversion.set_base :in
+    end
+  end
+
+  test "#add adds a new unit" do
+    @conversion.set_base :m
+    @conversion.add :in, aliases: [:inch], value: "0.0254 meter"
```

Comment:
would be slick to be able to omit the aliases array when there's a single imput:
`@conversion.add :in, aliases: :inch, value: "0.0254 meter"`